modules/test/playwright/pages/portal-workflow-kaleo-designer-web/ProcessBuilderPage.ts

```diff
@@ -25,6 +31,18 @@ export class ProcessBuilderPage {
 			.click();
 	}
 
+	async toggleSourceViewAndBack() {
```

Comment:
I think `switchToSourceViewAndBackToDiagram()` is a better name. Because this is such a crucial step to make sure things are working in Workflow, this deserves a dedicated method, like you did.

Reply:
Done!